docs/modules/mqtt.md

```diff
@@ -139,22 +139,6 @@ This is the description of how the `autoreconnect` functionality may (or may not
 > is considered a success if the client connects to a server and gets back a good response packet in response to its MQTT connection request.
 > This implies (for example) that the username and password are correct.
 
-#### Connection failure callback reason codes:
-
-| Constant | Value | Description |
-|----------|-------|-------------|
-|`mqtt.CONN_FAIL_SERVER_NOT_FOUND`|-5|There is no broker listening at the specified IP Address and Port|
-|`mqtt.CONN_FAIL_NOT_A_CONNACK_MSG`|-4|The response from the broker was not a CONNACK as required by the protocol|
-|`mqtt.CONN_FAIL_DNS`|-3|DNS Lookup failed|
-|`mqtt.CONN_FAIL_TIMEOUT_RECEIVING`|-2|Timeout waiting for a CONNACK from the broker|
-|`mqtt.CONN_FAIL_TIMEOUT_SENDING`|-1|Timeout trying to send the Connect message|
-|`mqtt.CONNACK_ACCEPTED`|0|No errors. _Note: This will not trigger a failure callback._|
-|`mqtt.CONNACK_REFUSED_PROTOCOL_VER`|1|The broker is not a 3.1.1 MQTT broker.|
-|`mqtt.CONNACK_REFUSED_ID_REJECTED`|2|The specified ClientID was rejected by the broker. (See `mqtt.Client()`)|
-|`mqtt.CONNACK_REFUSED_SERVER_UNAVAILABLE`|3|The server is unavailable.|
-|`mqtt.CONNACK_REFUSED_BAD_USER_OR_PASS`|4|The broker refused the specified username or password.|
-|`mqtt.CONNACK_REFUSED_NOT_AUTHORIZED`|5|The username is not authorized.|
-
```

Comment:
Hmm, I don't see any reference to these ever having existed in this module. I guess this was an esp8266 leftover?

Reply:
Yes, I found these reason codes only in the mqtt-8266 module.